docs/release-checklist.rst

```diff
@@ -115,8 +115,9 @@ they will need to evaluate which contributors' signatures they trust.
 
   - ``git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.16.0rc0" tahoe-lafs-1.16.0rc0``
 
-*Replace the key-id above with your own, which can simply be your email if's attached your fingerprint.*
-*Don't forget to put the correct tag message and name in this example the tag message is "release Tahoe-LAFS-1.16.0rc0" and the tag name is `tahoe-lafs-1.16.0rc0`* 
+.. note:: 
+    - Replace the key-id above with your own, which can simply be your email if's attached your fingerprint.
+    - Don't forget to put the correct tag message and name in this example the tag message is "release Tahoe-LAFS-1.16.0rc0" and the tag name is `tahoe-lafs-1.16.0rc0` 
 
```

Comment:
for consistency maybe double-quote the second (or at least fix the formatting to be valid rst) or code-quote the first